agent/provider/src/rules.rs

```diff
@@ -371,13 +368,9 @@ impl RulesManager {
         }
     }
 
-    fn verify_permissions(&self, cert: &str, demand_permissions_present: bool) -> Result<()> {
-        let mut required = vec![CertPermissions::OutboundManifest];
-
-        if demand_permissions_present {
-            required.push(CertPermissions::UnverifiedPermissionsChain);
-        }
-
+    //TODO delete when deleting X509 permissions feature
+    fn verify_permissions(&self, cert: &str) -> Result<()> {
```

Comment:
Why `demand_permisisons_present` was removed?

Reply:
Because there are no permissions arriving in demand - it was this permissions related demand property which was supposed to be used in future.